.vscode/launch.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
.vscode files should not be included in repo

Reply:
removed .vscode